client/src/model/data_layer.ts

```diff
@@ -121,15 +121,17 @@ enum MapLayer {
 }
 
 /**
- * Zoom levels for each demographic layer.
+ * Zoom thresholds for each geographic level.
```

Comment:
Does the enum applies if it's greater or less than the threshold? i.e. does `State` apply for 0 to 4, or 4 to 5?

(then could you add whatever the answer is to the comment here)

Reply:
Added a comment to clarify thanks